src/codeview.js

```diff
@@ -556,10 +579,28 @@ var Codeview = GObject.registerClass({
         }
     }
 
+    dbusRegister(appId, id) {
+        const objPath = Gio.Application.get_default().get_dbus_object_path();
+        const winName = appId.replace(/\./gi, '_');
+        this.objectPath = `${objPath}/window/${winName}/codeview/${id}`;
+        this._dbus = Gio.DBusExportedObject.wrapJSObject(CodeViewIface, this);
+        try {
+            this._dbus.export(Gio.DBus.session, this.objectPath);
+        } catch (e) {
+            logError(e);
```

Comment:
If we cannot export, wouldn't it make sense to set `this._dbus` to null here?

Reply:
yes, sure, I've just added that